### PR TITLE
chore(AvatarUpload): add remaining size variants

### PR DESCRIPTION
**Pull request by ozgurkececioglu**
[FX-3074]

### Description

- added remaining size variants from Avatar: `xxsmall`, `xsmall`, and `medium`
- fine-tuned AvatarUpload's border and outline shapes
- provide some more comments to ease understanding how DropzoneSvg works

### How to test

- go and check the sizes story https://picasso.toptal.net/fx-3073-avatar-upload-remaining-size-variants/?path=/story/components-avatarupload--avatarupload#sizes

### Screenshots

Screen Shot 2022-09-02 at 15 31 02

### Development checks

- [x] Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [x] Make sure that additions and changes on design follow [Toptal's BASE design](https://design.toptal.net/), and it's been already discussed with designers at #-base-core
- [x] Annotate all `props` in component with documentation
- [x] Create `examples` for component
- [x] Ensure that deployed demo has expected results and good examples
- [x] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [x] Self reviewed
- Covered with tests

**Breaking change**

- codemod is created and showcased in the changeset
- test alpha package of Picasso in StaffPortal

> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-3074]: https://toptal-core.atlassian.net/browse/FX-3074?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/picasso/src/AvatarUpload/DropzoneSvg/utils.ts [hunk 1]
```diff
@@ -1,106 +1,112 @@
...
+ *   x1,y1 ----------------- x2,y1
+ *     |                       |
+ *     |                       |
+ *     |                       |
+ *     |                       |
+ *     |                       |
+ *   x1,y3                     |
+ *      \                      |
+ *       \                     |
+ *       x3,y2-------------- x2,y2
```
Comment (augustobmoura) [inline packages/picasso/src/AvatarUpload/DropzoneSvg/utils.ts:20]: Nice drawing! :+1:

**Review by augustobmoura — APPROVED** [review]
The storybook examples look very good! Nice work :+1:!

File: packages/picasso/src/AvatarUpload/DropzoneSvg/utils.ts [hunk 2]
```diff
@@ -1,106 +1,112 @@
...
+ *
```
Comment (dmaklygin) [inline packages/picasso/src/AvatarUpload/DropzoneSvg/utils.ts:21]: 👍

File: packages/picasso/src/AvatarUpload/DropzoneSvg/utils.ts [hunk 3]
```diff
@@ -1,106 +1,112 @@
...
   const outlineStrokeWidth = 2 // width of the outline stroke
 
   return getAvatarShape({
     startPoint: centerShift,
     topEdgeLength: dimensions + outlineStrokeWidth,
     rightEdgeLength: dimensions + outlineStrokeWidth,
-    bottomEdgeLength: cornerSize + outlineStrokeWidth,
-    leftCornerPoint: dimensions - cornerSize + outlineStrokeWidth,
+    bottomEdgeStartPoint: cornerSize + outlineStrokeWidth + 1, // fine tuning for the border stroke
+    leftCornerEndPoint: dimensions - cornerSize + outlineStrokeWidth + 1, // fine tuning for the border stroke
```
Comment (dmaklygin) [inline packages/picasso/src/AvatarUpload/DropzoneSvg/utils.ts:95]: okay

**Review by dmaklygin — APPROVED** [review]
Code looks good.

**Review by OndrejTuma — APPROVED** [review]
Code looking good 👍

**Pull request merged**